herd/tests/instructions/AArch64/A43.litmus

```diff
@@ -0,0 +1,11 @@
+AArch64 A43
+(* Test load exclusive; Store exclusive, symbolic location *)
+
+ { 0:X4=z; z=2}
```

Comment:
Running this test on hardware resulted in a bus error. Probably because it writes a 64 bit wide value into a location (implicitly) declared to old a 32 bits wide integer.
```suggestion
 { 0:X4=z; int64_t z=2}
```